finality-provider/config/evm.go

```diff
@@ -10,18 +10,21 @@ const (
 )
 
 type EVMConfig struct {
-	RPCAddr string `long:"rpc-address" description:"address of the rpc server to connect to"`
+	RPCL1Addr                      string `long:"rpc-address" description:"address of the L2 RPC server to connect to"`
+	RPCL2Addr                      string `long:"rpc-address" description:"address of the L1 RPC server to connect to"`
+	L2OutputOracleContractAddress  string `long:"sol-address" description:"address of the L2output smart contract"`
+	BitcoinStackingContractAddress string `long:"sol-address" description:"address of the Bitcoinstaking smart contract"`
```

Comment:
why this is also called `long:"sol-address"`? sounds wrong

 what does "long" mean here?